controllers/topics/index.js

```diff
@@ -256,12 +261,13 @@ const _afterPutTopic = async (
   userId,
   name,
   isAnonymous,
-  withSystemMessage = false
+  withSystemMessage = false,
+  prevUserToken
 ) => {
   // follow / unfollow main feed topic
   try {
     if (isUnfollow) {
-      await unfollowMainFeedTopic(token, userId, name);
+      await unfollowMainFeedTopic(prevUserToken, userId, name);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `prevUserToken` as a parameter in the arrow function and its usage in `unfollowMainFeedTopic` function call seems fine. However, ensure that the called function has been updated to handle this new argument.
```diff
+   prevUserToken
+  ) => {
+    // follow / unfollow main feed topic
+    try {
+      if (isUnfollow) {
+       await unfollowMainFeedTopic(prevUserToken, userId, name);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->